data/kadi4mat_json/atmospheric_corrosion_models/din-corrosion-protection-model-iso-9223-compliant.json

```diff
@@ -0,0 +1,241 @@
+  "extras": [
+    {
+      "key": "Parameters",
+      "type": "dict",
+      "value": [
+        {
+          "key": "Time exponent",
+          "type": "list",
+          "value": [
+            {
+              "type": "str",
+              "value": "Use DIN recommended time exponents measured from the ISO CORRAG program"
+            },
+            {
+              "type": "str",
+              "value": "Enter manually"
+            }
+          ]
+        },
+        {
+          "key": "Corrosion type",
+          "type": "list",
+          "value": [
+            {
+              "type": "str",
+              "value": "Insignificant - Heated rooms with low relative humidity and insignificant air pollution, e.g., offices, schools, museums."
+            },
+            {
+              "type": "str",
+              "value": "Low - Unheated rooms with fluctuating temperature and relative humidity. Rare condensation and low air pollution, e.g., storage rooms, sports halls."
+            },
+            {
+              "type": "str",
+              "value": "Moderate - Rooms with occasional condensation and moderate air pollution from production processes, e.g., food processing plants, laundries, breweries, dairies."
+            },
+            {
+              "type": "str",
+              "value": "High - Rooms with frequent condensation and high air pollution from production processes, e.g., industrial plants, swimming pools."
+            },
+            {
+              "type": "str",
+              "value": "Very high - Rooms with very high frequency of condensation and/or high air pollution from production processes, e.g., mines, industrial cavities, non-ventilated halls in subtropical and tropical climate zones."
```

Comment:
Not sure if this will work probably we need to split the description from the value.

Reply:
Do list have description fields for entries?

Reply:
Not for individual elements, but for the whole list. I can put the descriptions there.